.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
   - sudo mv docker-compose /usr/local/bin
 
 script:
+  - sed -i "s/NODE_ENV=/NODE_ENV=$(echo $NODE_ENV)/" .env
```

Comment:
Sorry to be late to this --- what does this command do? I'm not experienced with `sed`.